Sources/DataStructures/BinaryHeap.swift

```diff
@@ -50,14 +50,6 @@ public struct BinaryHeap <Element: Hashable, Value: Comparable> {
         }
     }
     
-    /// Propose update of `element` to value `suggestion` (accept if `value(of: element)`
```

Comment:
Ack, this snuck into an unrelated commit.

This method is marked `internal`, but is not used by anything internally. There was an `@testable import` in the `BinaryHeapTests` to test this.

@bwetherfield, should this be made `public`, destroyed, or something else?